pkgs/top-level/all-packages.nix

```
@@ -15588,7 +15588,7 @@ in
 
   libclxclient = callPackage ../development/libraries/libclxclient  { };
 
-  libco-canonical = callPackage ../development/libraries/libco-canonical { };
+  libco-canonical = throw "libco-canonical removed since 2021-05-16: Canonical deleted the repo, libco-canonical is not used anymore.";
```

```suggestion
  libco-canonical = throw "libco-canonical: Canonical deleted the repo, libco-canonical is not used anymore."; # added 2021-05-16
```

and move that line to aliases.nix.

---

Ah, TDIL. I wasn't aware of this convention. Thank you for letting me know. I updated the PR accordingly.